schemas/deviceToCloud/ground_fix/ground-fix.json

```diff
@@ -123,7 +143,8 @@
     "definitions": {
         "DoReply": {
             "type": "boolean",
-            "description": "Does not reply, even in event of an error, if set to false. Defaults to true."
+            "description": "Controls the response body. If boolean false or number 0, response body will be empty.",
```

Comment:
do all `"type": "boolean"` values in the cloud schemas also accept numeric `1`/`0` values?

Reply:
This is a good point. The schema is actually wrong saying that doReply type is boolean. It should be `type: ['boolean', 'integer']. Will update.

Reply:
... and the new config flags are all booleans.

Reply:
is there extra overhead to making them boolean and integer?

Reply:
Extra overhead in the sense that it adds complexity having to check for both values instead of just one, yes. 

Does integer work better, payload size wise? If the new config was `fallback: 0` instead of `fallback: false`, does that make a huge difference?

Reply:
again, not a huge difference... just a few bytes.  in the grand scheme of things, probably insignificant.

but we should be conscious of payload sizes when developing these JSON messages that are sent to/from devices.

Reply:
Agreed. My goal is to keep the format between the REST and MQTT APIs the same. It reduces complexity significantly when both endpoints can use the same logic.